src/cryptoadvance/specter/cli/cli_bitcoind.py

```diff
@@ -0,0 +1,242 @@
+            try:
+                # Get process name & pid from process object.
+                processName = proc.name()
+                pid = proc.pid
+                if processName.startswith("bitcoind"):
+                    echo(f"Killing bitcoind-process with id {pid} ...")
+                    did_something = True
+                    os.kill(pid, signal.SIGTERM)
+            except (psutil.NoSuchProcess, psutil.AccessDenied, psutil.ZombieProcess):
+                echo(f"Pid {pid} not owned by us. Might be a docker-process? {proc}")
+        if Path(data_dir).exists():
+            echo(f"Purging Datadirectory {data_dir} ...")
+            did_something = True
+            shutil.rmtree(data_dir)
+        if not did_something:
+            echo("Nothing to do!")
+        return
+    mining_every_x_seconds = float(mining_period)
+    if nodocker:
+        echo("starting plain bitcoind")
+        my_bitcoind = BitcoindPlainController()
+        # Make sure datadir does exist if specified:
+        Path(data_dir).mkdir(parents=True, exist_ok=True)
+    else:
+        echo("starting or detecting container")
+        my_bitcoind = BitcoindDockerController(docker_tag=docker_tag)
+    try:
+        my_bitcoind.start_bitcoind(cleanup_at_exit=True, datadir=data_dir)
+    except docker.errors.ImageNotFound:
+        echo(f"Image with tag {docker_tag} does not exist!")
+        echo(
+            f"Try to download first with docker pull \
+                     registry.gitlab.com/cryptoadvance/specter-desktop\
+                     /python-bitcoind:{docker_tag}"
+        )
+        sys.exit(1)
+    if not nodocker:
+        tags_of_image = [
+            image.split(":")[-1] for image in my_bitcoind.btcd_container.image.tags
+        ]
+        if docker_tag not in tags_of_image:
+            echo(
+                "The running docker container is not \
+                                the tag you requested!"
+            )
+            echo(
+                "please stop first with docker stop {}".format(
+                    my_bitcoind.btcd_container.id
+                )
+            )
+            sys.exit(1)
+        echo("containerImage: %s" % my_bitcoind.btcd_container.image.tags)
+    echo("           url: %s" % my_bitcoind.rpcconn.render_url())
+    echo("user, password: bitcoin, secret")
+    echo("    host, port: localhost, 18443")
+    echo(
+        "   bitcoin-cli: bitcoin-cli -regtest -rpcuser=bitcoin -rpcpassword=secret getblockchaininfo "
+    )
+
+    if create_conn_json:
+        conn = my_bitcoind.rpcconn.as_data()
+        conn["pid"] = os.getpid()  # usefull to sen signals
+        with open("btcd-conn.json", "w") as file:
+            file.write(json.dumps(conn))
+
+        def cleanup():
+            os.remove("btcd-conn.json")
+
+        atexit.register(cleanup)
+
+    signal.signal(
+        signal.SIGUSR1,
+        lambda x, y: mine_2_specter_wallets(
+            my_bitcoind, config_obj["DATA_FOLDER"], echo
```

Comment:
```suggestion
            my_bitcoind, config_obj["SPECTER_DATA_FOLDER"], echo
```

Should be `SPECTER_DATA_FOLDER ` I think...